util/Migrator/DbScripts/2020-12-14_00_Permissions.sql

```diff
@@ -0,0 +1,544 @@
+IF COL_LENGTH('[dbo].[OrganizationUser]', 'AccessBusinessPortal') IS NULL
+BEGIN
+    ALTER TABLE
+        [dbo].[OrganizationUser]
+    ADD
+        [AccessBusinessPortal] BIT NULL
+END
+GO
+
+IF COL_LENGTH('[dbo].[OrganizationUser]', 'AccessEventLogs') IS NULL
+BEGIN
+    ALTER TABLE
+        [dbo].[OrganizationUser]
+    ADD
+        [AccessEventLogs] BIT NULL
+END
+GO
+
+IF COL_LENGTH('[dbo].[OrganizationUser]', 'AccessImportExport') IS NULL
+BEGIN
+    ALTER TABLE
+        [dbo].[OrganizationUser]
+    ADD
+        [AccessImportExport] BIT NULL
+END
+GO
+
+IF COL_LENGTH('[dbo].[OrganizationUser]', 'AccessReports') IS NULL
+BEGIN
+    ALTER TABLE
+        [dbo].[OrganizationUser]
+    ADD
+        [AccessReports] BIT NULL
+END
+GO
+
+IF COL_LENGTH('[dbo].[OrganizationUser]', 'ManageAllCollections') IS NULL
+BEGIN
+    ALTER TABLE
+        [dbo].[OrganizationUser]
+    ADD
+        [ManageAllCollections] BIT NULL
+END
+GO
+
+IF COL_LENGTH('[dbo].[OrganizationUser]', 'ManageAssignedCollections') IS NULL
+BEGIN
+    ALTER TABLE
+        [dbo].[OrganizationUser]
+    ADD
+        [ManageAssignedCollections] BIT NULL
+END
+GO
+
+IF COL_LENGTH('[dbo].[OrganizationUser]', 'ManageGroups') IS NULL
+BEGIN
+    ALTER TABLE
+        [dbo].[OrganizationUser]
+    ADD
+        [ManageGroups] BIT NULL
+END
+GO
+
+IF COL_LENGTH('[dbo].[OrganizationUser]', 'ManagePolicies') IS NULL
+BEGIN
+    ALTER TABLE
+        [dbo].[OrganizationUser]
+    ADD
+        [ManagePolicies] BIT NULL
+END
+GO
+
+IF COL_LENGTH('[dbo].[OrganizationUser]', 'ManageUsers') IS NULL
+BEGIN
+    ALTER TABLE
+        [dbo].[OrganizationUser]
+    ADD
+        [ManageUsers] BIT NULL
+END
+GO
+
+UPDATE [dbo].[OrganizationUser] 
```

Comment:
So, this is going to get expensive, however, you need to do the following:
1. Remove the `GO` statements between this `UPDATE` statement and the `ALTER COLUMN` statements
2. Wrap that whole block in a `TRANSACTION` so it "blocks" modification to prevent errors/issues with insertions happening before/after each of the statements

Reply:
Another benefit of converting to a json blob: no updates to existing records need to be made anymore, we just do null checks and whatnot in the application logic. So, I guess this one is fixed.